items/SCATHA;4.json

```diff
@@ -17,12 +17,12 @@
     "§7find treasure while mining.",
     "",
     "§6Wormhole",
-    "§7§7Gives a §a{2}% §7chance to mine 2",
+    "§7§7Gives a §a{2}% §7to mine 2",
     "§7adjacent stone or hard stone.",
     "",
     "§6Gemstone Power",
-    "§7§7Gain §a+{3}% §7more Gemstone",
-    "§7Powder.",
+    "§7§7Gain §a+{3}% §7more §dGemstone",
+    "§7Powder§7.",
```

Comment:
That makes no sense

Reply:
Cause Hypixel changed the pet's lore to now make the Gemstone Powder pink colour

Reply:
> Cause Hypixel changed the pet's lore to now make the Gemstone Powder pink colour

Yes but you still do §7 at the start of the second line and then you do §7 at the end again.

Reply:
was a typo oops